packages/components/psammead-live-label/package.json

```diff
@@ -1,13 +1,13 @@
 {
   "name": "@bbc/psammead-live-label",
-  "version": "2.0.26",
+  "version": "2.0.27",
   "main": "dist/index.js",
   "module": "esm/index.js",
   "sideEffects": false,
   "description": "React component for a live label",
   "repository": {
     "type": "git",
-    "url": "https://github.com/BBC-News/psammead/tree/latest/packages/components/psammead-social-embed"
+    "url": "https://github.com/BBC-News/psammead/tree/latest/packages/components/psammead-live-label"
```

Comment:
Was I correct in updating the git url. Was this deliberate or just missed out?

Reply:
```suggestion
    "url": "https://github.com/bbc/psammead/tree/latest/packages/components/psammead-live-label"
```